addons/mail/static/src/models/model/model.js

```diff
@@ -189,6 +179,18 @@ function factory() {
             return _.uniqueId(`${this.modelName}_`);
         }
 
+        /**
+         * This function is called after the record has been created, more
+         * precisely at the end of the update cycle (which means all implicit
+         * changes such as computes have been applied too).
+         *
+         * The main use case is to register listeners on the record.
+         *
+         * @abstract
+         * @private
+         */
+        _created() {}
+
```

Comment:
Convention is widget/components is to have lifecycle methods in their own category (or at a similar level as constructor). Also they are ordered by lifecycle, so `willCreated` => `created` => `deleted`.

`_updateAfter`/`_updateBefore` are "bad" hooks, so I would let them be and wish for their extinction in the near future.

Reply:
ok